utoipa-swagger-ui/Cargo.toml

```diff
@@ -34,3 +34,6 @@ rustdoc-args = ["--cfg", "doc_cfg"]
 [build-dependencies]
 zip = { version = "0.6", default-features = false, features = ["deflate"] }
 regex = "1.7"
+reqwest = { version = "0.11", features = ["blocking"] }
+openssl = { version = "0.10", features = ["vendored"] }
+anyhow = "1.0"
```

Comment:
Probably overkill to add anyhow error for just one error, instead I would maybe just return `Box<&dyn Error>` it is only a build code and there is necessity to know what the error is. :point_down:

Reply:
Yes I full agree. I changed it. Cf. commit.